scri/asymptotic_bondi_data/bms_charges.py

```diff
@@ -99,6 +101,37 @@ def bondi_angular_momentum(self, output_dimensionless=False):
     return charge_vector_from_aspect(charge_aspect)[:, 1:]
 
 
+def CWWY_angular_momentum(self):
+    """Compute the Chen/Wang/Wang/Yau angular momentum vector.
+
+    See Eq. (5) of https://arxiv.org/abs/2102.03235
+
+    """
+    ell_max = 1  # Compute only the parts we need, ell<=1
+    supertranslation_potential_data = scri.asymptotic_bondi_data.map_to_superrest_frame.𝔇inverse(
+        np.array(self.sigma.ethbar_GHP.ethbar_GHP + self.sigma.bar.eth_GHP.eth_GHP), self.ell_max
```

Comment:
Is it faster to just compute `self.sigma.ethbar_GHP.ethbar_GHP` and take twice the real part?